GitDiffMargin/Git/GitCommands.cs

```diff
@@ -47,27 +39,14 @@ public IEnumerable<HunkRangeInfo> GetGitDiffFor(ITextDocument textDocument, stri
                     yield break;
 
                 var retrieveStatus = repo.RetrieveStatus(originalPath);
-                if (retrieveStatus == FileStatus.Nonexistent)
-                {
-                    // this occurs if a file within the repository itself (not the working copy) is opened.
```

Comment:
📝 Comment handling in whatever tool formatted this method is a serious problem. It's severe enough that I would recommend reporting this as a bug to the tool vendor.